src/stores/messages/SavedMessageViewType.ts

```diff
@@ -28,25 +28,27 @@ export class SavedMessageViewType {
 		this.message = message;
 
 		this.messageDisplayRulesStore = messageDisplayRulesStore;
-		reaction(() => this.displayRule, this.setViewType);
+
```

Comment:
we don't need reaction now to displayRule? this makes MessagesViewConfigurator component almose useless